src/config/dark-sites.config

```diff
@@ -197,6 +197,7 @@ daily-fire.com
 daksh.eu.org
 danidev.net
 dankmemer.gg
+danstudioapps.com
```

Comment:
![image](https://user-images.githubusercontent.com/25481501/148246274-dd257b2f-463d-47c8-95cf-f5374b7f621d.png)
This isn't that dark to me.